packages/stream_chat_flutter/example/pubspec.yaml

```diff
@@ -13,30 +27,53 @@ dependencies:
     path: ../
   stream_chat_persistence:
     path: ../../stream_chat_persistence
```

Comment:
These relative paths should be removed.

Reply:
Are you sure?
I probably agree on the persistence package, but why should we remove the relative import of the sdk?

Reply:
Flutter is usually installed at `sdk: flutter`, not `path: ../`. A user may not have the framework installed in that directory.

Reply:
expand the diff :)